tests/utils.go

```diff
@@ -3364,16 +3364,20 @@ func CreateISCSITargetPOD(containerDiskName cd.ContainerDisk) (iscsiTargetIP str
 	}
 
 	pod, err = virtClient.CoreV1().Pods(NamespaceTestDefault).Create(pod)
-	PanicOnError(err)
+	Expect(err).ToNot(HaveOccurred(), "should successfully create ISCSI target pod")
 
 	getStatus := func() k8sv1.PodPhase {
 		pod, err := virtClient.CoreV1().Pods(NamespaceTestDefault).Get(pod.Name, metav1.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
-		iscsiTargetIP = pod.Status.PodIP
+		Expect(err).NotTo(HaveOccurred(), "should fetch ISCSI target pod for 'Status.Phase' checking")
 		return pod.Status.Phase
 	}
-	Eventually(getStatus, 120, 1).Should(Equal(k8sv1.PodRunning))
-	return
+	Eventually(getStatus, 120, 1).Should(Equal(k8sv1.PodRunning), "should fetch running ISCSI target pod")
```

Comment:
Would you consider having this available as a public function on a library ?

I think having 1 function that receives the name of a pod (along w/ namespace), plus the 'desired' state, and returns a boolean stating if the desired state is met would be very helpful. 

You could then wrap that up in a private function here (lambda would do) that would have the `Eventually(<the function stated above>).To(BeTrue()) ` . 

But, not a blocker.